pkg/ebpf/telemetry/errors_telemetry.go

```diff
@@ -117,6 +118,33 @@ func (e *ebpfTelemetry) fill(maps []names.MapName, mn names.ModuleName, mapErrMa
 	return nil
 }
 
+func (e *ebpfTelemetry) cleanup(maps []names.MapName, mn names.ModuleName, mapErrMap *maps.GenericMap[uint64, mapErrTelemetry], helperErrMap *maps.GenericMap[uint64, helperErrTelemetry]) error {
+	e.mtx.Lock()
+	defer e.mtx.Unlock()
+
+	// Cleanup mapKeys (initialized in initializeMapErrTelemetryMap)
+	h := keyHash()
+	for _, mapName := range maps {
+		delete(e.mapKeys, mapTelemetryKey(mapName, mn))
```

Comment:
why not just create a new empty map instead of deleting each element?
`e.mapKeys = make(map[telemetryKey]uint64)`